hystrix/src/main/java/feign/hystrix/HystrixFeign.java

```diff
@@ -79,137 +76,26 @@ public InvocationHandler create(Target target, Map<Method, MethodHandler> dispat
      *   }
      * };
      *
-     * GitHub github = HystrixFeign.builder()
+     * GitHub github = HystrixFeign.builder(fallback)
      *                             ...
-     *                             .target(GitHub.class, "https://api.github.com", fallback);
+     *                             .target(GitHub.class, "https://api.github.com");
      * }</pre>
-     *
-     * @see #target(Target, Object)
      */
     public <T> T target(Class<T> apiType, String url, T fallback) {
       return target(new Target.HardCodedTarget<T>(apiType, url), fallback);
     }
 
-    /**
-     * @see feign.Feign.Builder#contract
-     */
-    public Builder contract(Contract contract) {
-      this.contract = contract;
-      return this;
-    }
-
-    /**
-     * @see feign.Feign.Builder#build
-     */
-    public Feign build() {
-      delegate.invocationHandlerFactory(new HystrixInvocationHandler.Factory());
-      delegate.contract(new HystrixDelegatingContract(contract));
-      return delegate.build();
-    }
-
-    // re-declaring methods in Feign.Builder is same work as covariant overrides,
-    // but results in less complex bytecode.
-
-    /**
-     * @see feign.Feign.Builder#target(Class, String)
-     */
-    public <T> T target(Class<T> apiType, String url) {
-      return target(new Target.HardCodedTarget<T>(apiType, url));
-    }
-
-    /**
-     * @see feign.Feign.Builder#target(Target)
-     */
-    public <T> T target(Target<T> target) {
-      return build().newInstance(target);
-    }
-
-    /**
-     * @see feign.Feign.Builder#logLevel
-     */
-    public Builder logLevel(Logger.Level logLevel) {
-      delegate.logLevel(logLevel);
-      return this;
-    }
-
-    /**
-     * @see feign.Feign.Builder#client
-     */
-    public Builder client(Client client) {
-      delegate.client(client);
+    @Override
+    public Feign.Builder invocationHandlerFactory(InvocationHandlerFactory invocationHandlerFactory) {
+      //TODO: throw exception, log warning or do nothing?
```

Comment:
Since the method can no longer be hidden, what do you think should be done here?